packages/home-spa/package.json

```diff
@@ -32,7 +32,7 @@
     "css-loader": "3.4.2",
     "file-loader": "6.0.0",
     "html-loader": "1.0.0",
-    "html-webpack-plugin": "3.2.0",
+    "html-webpack-plugin": "4.0.0",
```

Comment:
Can you make it up to `^4.0.0`

So it will download the latest minor version whenever available.

Reply:
I will do that, but the renovate bot will make a PR to pin the dependency @deshmukhmayur

Reply:
We can discard that PR. We don't need multiple PR for the same fix.

Reply:
Then we will have to discard this PR https://github.com/1-Platform/one-platform/pull/117